cmd/soroban-cli/src/commands/snapshot/create.rs

```diff
@@ -55,24 +55,27 @@ fn default_out_path() -> PathBuf {
 ///
 /// Filters (address, wasm-hash) specify what ledger entries to include.
 ///
-/// Account addresses include the account, and trust lines.
+/// Account addresses include the account, and trustlines.
```

Comment:
Trust lines are actually two words, not one.
```suggestion
/// Account addresses include the account, and trust lines.
```

Reply:
The docs always used "trustlines". 🤔 

https://developers.stellar.org/docs/learn/fundamentals/stellar-data-structures/accounts#trustlines

Reply:
In the xdr we use trust line. We've always been inconsistent with use in the docs, although it looks recently like there are now more references to trustlines rather than trust lines. I guess it doesn't really matter. 🤷🏻